### PR TITLE
Add v1.14.0 release blog post

### PR DESCRIPTION
**Pull request by pmuens**
**Todos:**

- [x] Write content for each headline / feature
- [x] Update contributors list

### CODE REVIEW

File: posts/2017-05-24-serverless-v1.14.0.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+Serverless will anaylze your `serverless.yml` file and create a dedicated nested stack for each function and their related resources.
+
+You won't notice any difference using this feature since Serverless will treat your `serverless.yml` file as is without any further modification. However behind the scenes the deployment process will be different since your whole service will be deployed with multiple stacks.
+
+**Important:** This feature is still experimental. It was tested with different, really complex services throughout development. However there might be still some edge-cases where it could potentially uncover some bugs. Furthermore this feature is a one-way road. Once enabled it's pretty hard to go back to a single stack deployment. If you're looking for more control of the stack / service splitting process you might want to check out our support for ["Cross Service Communication"](https://serverless.com/framework/docs/providers/aws/guide/variables#reference-cloudformation-outputs) or the AWS intrinsic function `Fn::ImportValue`.
+
+### Login command
+
+- https://github.com/serverless/serverless/pull/3558
```
Comment (worldsoup) [inline posts/2017-05-24-serverless-v1.14.0.md]: CC @DavidWells

**Comment by pmuens** [comment]
Just updated the content so that it covers all the features we're about to release.
Maybe we can add some more content / context for the `login` functionality (right now it's just a rough description).

Additionally we might want to re-think if the Google Cloud Function announcement should be part of this blog post or posted separately (as @worldsoup pointed out yesterday).

I'll update the contributors list as soon as the milestone is closed.

/cc @worldsoup @DavidWells

**Comment by pmuens** [comment]
Ok. Updated everything and checked it twice.

It's good to go from my side @worldsoup @DavidWells

File: posts/2017-05-24-serverless-v1.14.0.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+It's recommended to issue a `serverless deploy` with the updated / fixed code some time after rolling back to put the service into a stable state again.
+
+**Note:** Lambda versioning needs to be enabled to use this feature. Serverless automatically adds versioning support for your Lambda functions unless you've opted out. See [our docs](https://serverless.com/framework/docs/providers/aws/guide/functions#versioning-deployed-functions) for more information on Lambda versioning.
+
+### DeadLetterConfig support
+
+With Serverless v1.14 you can use `DeadLetterConfig` support natively in Serverless.
+
+We've implemented the `onError` config parameter which you can sepcify on a function level. All you need to do is plug in your `SNS` topic `arn` and re-redploy your stack.
```
Comment (j0k3r) [inline posts/2017-05-24-serverless-v1.14.0.md]:
```diff
-sepcify
+specify
```

Comment (j0k3r) [inline posts/2017-05-24-serverless-v1.14.0.md]: Also, is _re-redploy_ means re-deploy or re-redeploy?

File: posts/2017-05-24-serverless-v1.14.0.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+
+**Note:** Serverless currently only supports `sns` topic arns due to a race condition when using SQS queue arns and updating the permissions in the IAM role. We're currently looking into this so that we can add support for SQS topic arns soon!
+
+### Login command
+
+In v1.14 we're introducing the `login` and `logout` commands for the Serverless CLI.
+
+You can re-use your Serverless account credentials you've obtained by [signing up for our beta](https://serverless.com/) program.
+
+After signing in Serverless will help you with some tips and tricks to further enhance you development workflow.
```
Comment (j0k3r) [inline posts/2017-05-24-serverless-v1.14.0.md]:
```diff
-you development
+your development
```

File: posts/2017-05-24-serverless-v1.14.0.md [hunk 4]
```diff
@@ -0,0 +1,236 @@
+---
+title: Google Cloud Functions support added in Serverless v1.14 deploy your functions to the Google Cloud
```
Comment (DavidWells) [inline posts/2017-05-24-serverless-v1.14.0.md]: "Serverless v1.14 - Deploy to Google Cloud Functions & rollback function support added"

Title tweak suggestion

cc @worldsoup

**Pull request merged**